diff --git a/dlls/winex11.drv/x11drv_main.c b/dlls/winex11.drv/x11drv_main.c
index b14d4d1af6..1e9e10fd3b 100644
--- a/dlls/winex11.drv/x11drv_main.c
+++ b/dlls/winex11.drv/x11drv_main.c
@@ -205,7 +205,8 @@ static inline BOOL ignore_error( Display *display, XErrorEvent *event )
     if ((event->request_code == X_SetInputFocus ||
          event->request_code == X_ChangeWindowAttributes ||
          event->request_code == X_ConfigureWindow ||
-         event->request_code == X_SendEvent) &&
+         event->request_code == X_SendEvent ||
+         event->request_code == X_UnmapWindow) &&
         (event->error_code == BadMatch ||
          event->error_code == BadWindow)) return TRUE;
 
